website/static/js/pages/project-dashboard-page.js

```diff
@@ -145,17 +145,15 @@ $(document).ready(function () {
                 });
             });
         },
-        onRemoveTag: function(tag){
+        onRemoveTag: function(tag) {
             var url = nodeApiUrl + 'tags/';
-            var data = JSON.stringify({tag: tag});
-            var request = $.ajax({
-                url: url,
-                type: 'DELETE',
-                contentType: 'application/json',
-                dataType: 'JSON',
-                data: data
-            });
+            // Don't try to delete a blank tag (would result in a server error)
+            if (!tag) {
+                return false;
+            }
+            var request = $osf.ajaxJSON('DELETE', url, {'data': {'tag': tag}});
             request.fail(function(xhr, textStatus, error) {
+                $osf.growl('Error', 'Could not remove tag');
```

Comment:
I'm not sure if this is better. Currently, if two people have the same project page open, and one person removes a tag, followed by another person removing the same tag on their browser, they get an error. Technically, no, the second person couldn't remove the tag, but that's just because the tag was gone. From the user's standpoint, the tag is gone, so everything is good.